### ISSUE TITLE
Image Annotation - Possible Error

### ISSUE THREAD

**Issue by yiping-wang** [paip-2019/challenge#25]
Sorry for opening two issues on the same day :)

Please check the below picture. The left shows a patch from the `Training_phase_1_001`; the middle shows the corresponding ground truth mask; the right shows the overlay between the patch and the mask. 

![whole](https://user-images.githubusercontent.com/46113612/58068704-3eeb1900-7b47-11e9-8567-a83bdaf768cd.png)

As you may observe, although the patch only contains one tissue (or cell), the ground truth mask labels multiple spots for tumour tissues (or cells). These spots are nearly indiscernible by humans. Also, the annotations are inconsistent. For example, we can observe there is a pink line on the left upside corner, and the mask only has labels for the very top part of the line.

Therefore, are these annotations possible errors or we can just ignore these details? I have a feeling this might not be important but I just want to ask for your confirmation. 

Thanks for answering :)

You can reproduce the above mask using the following code:

```python
mask_tif = mask_loader('~/Desktop/paip2019/Training_phase_1_001/01_01_0083_whole.tif')
# I also convert 1 to 255 in mask_loader for better viewing;
mask = mask_tif[72*512:73*512, 58*512:59*512]
mask_img = Image.fromarray(mask)
mask_img.save('mask_img.png')
```
A similar code for the WSI can be produced easily. :)

**Comment by hjoonjang** [comment]
@yiping-wang 
Thank you for your active participation!
Overall, it is true that those noises are inevitable and you can either just ignore or process them by your own arbitrary criteria. (For processing them, you might be interested in morphology libraries.)

We actually have seen a lot of examples like this as well as all the other researchers do in the field of digital histopathology, but it is hard to tell which version of preprocessing is clearly the best among the variants unless we adopt a fancy algorithm dedicated for this. That's why we just picked a possible method and share the details of it. Of course, you can implement something better than this for preprocessing ground-truth from the XML files, which, on the other hand, may possibly improve the performance of your algorithm eventually. Additionally, I would say the ground-truth for the validation/test data will be much cleaner than the training dataset.
